src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -128,11 +131,11 @@ public void setMaxOutput(double maxOutput) {
     this.differentialDrive.setMaxOutput(maxOutput);
```

Comment:
Note that this does not limit the output power if we access `leftGearbox` or `rightGearbox` directly, so anything that bypasses `differentialDrive` is not limited here.